internal/variant/variant.go

```diff
@@ -76,7 +127,36 @@ func FromString(oid string) (Variant, error) {
 	return nil, fmt.Errorf("unknown OID: %q", oid)
 }
 
-// Dummy OID for testing.
+// ValidProvider returns true if the attestation type is valid for the given provider.
+func ValidProvider(provider cloudprovider.Provider, variant Variant) bool {
+	validTypes, ok := providerAttestationMapping[provider]
+	if ok {
+		for _, aType := range validTypes {
+			if variant.Equal(aType) {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// Default is the default variant for the cloud provider.
+type Default struct{}
+
+// OID returns the struct's object identifier.
+func (Default) OID() asn1.ObjectIdentifier {
+	return asn1.ObjectIdentifier{1, 3, 9900, 0, 0}
+}
```

Comment:
This variant is basically just a copy of `Dummy`.
I think it can be removed and replaced with `Dummy` in the places where it is currently used.